bgpd/bgp_vty.c

```diff
@@ -1411,6 +1411,36 @@ DEFUN (no_bgp_cluster_id,
 	return CMD_SUCCESS;
 }
 
+DEFPY (bgp_norib, bgp_norib_cmd, "bgp no-rib", BGP_STR
```

Comment:
Please follow the style for DEFUN/DEFPY's around this new code.  Each parameter should get it's own line

Reply:
I have been getting conflicting instructions regarding the style convention in the use of DEFSH/DEFUN/DEFPY/DEFPY_YANG macros. Some devs tell me to use one line per parameter, others tell me to fill the line space. Is there a documented TSC decision on this?